### PR TITLE
Add aws config in helm values

### PR DESCRIPTION
**Pull request by kartik-moolya**
##### ISSUE TYPE

 - Feature Pull Request

##### SUMMARY

adding config values for helm deployments

adding config values for helm deployments

Fixes #XXX

### CODE REVIEW

**Comment by PrasadG193** [comment]
@kartik-moolya please also update deploy-all-in-one* yamls

**Comment by kartik-moolya** [comment]
> @kartik-moolya please also update deploy-all-in-one* yamls

File: deploy-all-in-one-tls.yaml [hunk 1]
```diff
@@ -258,17 +258,20 @@ stringData:
...
       elasticsearch:
-        enable: false
-        server: 'ELASTICSEARCH_ADDRESS'              # e.g https://example.com:9243
-        username: 'ELASTICSEARCH_USERNAME'
+        enabled: false
+        awsSigning:
+          enabled: false                             # enable awsSigning using IAM for Elastisearch hosted on AWS, if true make sure AWS environment variables are set. Refer https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-envvars.html
+          awsRegion: "us-east-1"                     # AWS region where Elasticsearch is deployed
+        server: 'ELASTICSEARCH_ADDRESS'                                    # e.g https://example.com:9243
+        username: 'ELASTICSEARCH_USERNAME'                                 # Basic Auth
```
Comment (PrasadG193) [inline deploy-all-in-one-tls.yaml:266]: Please align comments at the same level. Remove extra spaces

File: deploy-all-in-one.yaml [hunk 2]
```diff
@@ -258,17 +258,20 @@ stringData:
...
       elasticsearch:
-        enable: false
-        server: 'ELASTICSEARCH_ADDRESS'              # e.g https://example.com:9243
-        username: 'ELASTICSEARCH_USERNAME'
+        enabled: false
+        awsSigning:
+          enabled: false                             # enable awsSigning using IAM for Elastisearch hosted on AWS, if true make sure AWS environment variables are set. Refer https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-envvars.html
+          awsRegion: "us-east-1"                     # AWS region where Elasticsearch is deployed
+        server: 'ELASTICSEARCH_ADDRESS'                                    # e.g https://example.com:9243
+        username: 'ELASTICSEARCH_USERNAME'                                 # Basic Auth
```
Comment (PrasadG193) [inline deploy-all-in-one.yaml:266]: Same here and other places

File: deploy-all-in-one-tls.yaml (hunk 1 above)
Comment (kartik-moolya) [reply deploy-all-in-one-tls.yaml]: done

File: deploy-all-in-one.yaml (hunk 2 above)
Comment (kartik-moolya) [reply deploy-all-in-one.yaml]: done

**Pull request merged**